### DISCUSSION TITLE
Unexpected closure

### DISCUSSION THREAD

**Discussion by pollcastillo** [rsayfutdinov/Pikka#17] [Q&A] [answered]
Sorry, my English is too bad.
When I try to select a color the application closes. I use macOS Catalina 10.15.7.

**Comment by rsayfutdinov** [comment, marked as answer]
> Sorry, my English is too bad.
> 
> When I try to select a color the application closes. I use macOS Catalina 10.15.7.

I apologize that you have issues using the App. Unfortunately, macOS 10.15.7 is not supported anymore. The app requires at least macOS 11. 

Best regards, 
Ruslan.
